src/odemis/acq/stitching/_registrar.py

```diff
@@ -648,10 +651,10 @@ def _get_shift(self, prev_tile, tile):
         # Normalized cross correlation (values between -1 and 1)
         ncc = (covar / (stDev[0] * stDev[1]))
 
-        overlap = min(numpy.abs(numpy.subtract(tile.shape, numpy.abs(exp_shift))))
-        if max(numpy.abs(shift)) > overlap:
-            logging.info("Calculated shift is larger than the overlap size, using expected position "
-                         "instead.")
+        overlap = numpy.abs(numpy.subtract(tile.shape[::-1], numpy.abs(exp_shift)))
```

Comment:
I would add a comment here explaining the .shape[::-1] to flip the xy dimenisons